openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx

```diff
@@ -513,9 +505,10 @@ const EntityPageInfo = ({
                 position="bottom-end"
                 theme="light"
                 onRequestClose={() => setShowActions(false)}>
-                <span>
-                  <FontAwesomeIcon icon="ellipsis-vertical" />
-                </span>
+                <FontAwesomeIcon
+                  className="tw-text-primary"
+                  icon="ellipsis-vertical"
+                />
```

Comment:
Can we replace it with AntD? So that in future we can remove fontAwesome lib?

Reply:
@chirag-madlani, Sure, I will have a look into this.